### ISSUE TITLE
Upgrading CISCO SDK from 2.8 to 3.6.0/3.7.0 - While on going call if we minimize app and again open then Remote Video disappear

### ISSUE THREAD

**Issue by CDL24** [webex/webex-android-sdk#130]
Below is the use cases : 
1. Join call
2. Now when call is on going just minimize App and again open RemoteView will be disappear

Second Use case :

1. When call is on going and If you open another activity or screen and again come back to Video calling screen then RemoteView will be disappear.

This both issue are still observable in 3.6.0 and 3.7.0 KintchenSink apk 

Kindly suggest the solution for the same as this Video calling is important feature

**Comment by Rudsriva** [comment]
Can you provide us the steps on how you're reproducing this issue? Also, supporting screenshots or a code snippet would help us to further investigate.
Could you try reproducing this issue with our demo KitchenSink App: https://github.com/webex/webex-android-sdk-example if not done already? 
If you're able to reproduce the issue on our demo app, kindly let us know the steps you followed from scratch.

**Comment by CDL24** [comment]
Yes I have tested both 3.6.0 and 3.7.0 KitchenSink app apk downloaded from webex git page. In both apk this issue is reproducible.
Steps:
1. Join call in 2 device 
2. Now Minimize app and maximize again in any one device then RemoteVideo will be disappear.

**Comment by CDL24** [comment]
is there any update on this ?

**Comment by Stackflood** [comment]
@CDL24 , we are not able to even start the app after the upgrade. Its crashing and showing this issue : https://github.com/webex/webex-android-sdk/issues/127

Can you please help ?
